src/module.c

```diff
@@ -52,6 +52,8 @@ JL_DLLEXPORT jl_module_t *jl_new_module_(jl_sym_t *name, jl_module_t *parent, ui
     m->compile = -1;
     m->infer = -1;
     m->max_methods = -1;
+    m->file = name; // Using the name as a placeholder is better than nothing
```

Comment:
Is it though? It seems slightly obnoxious to make the user guess if it is a real file or a placeholder (vs. using something like empty_sym here)

Reply:
I'm ok with empty_sym. Seemed slightly nicer to the end user if printing would at least print the module name, but probably that's a printing concern and shouldn't be done here.

Reply:
That was my thought, since the main use case is making Revise work better

Alternatively, we could put `jl_symbol("nonE")` here, which is what the other code path would do if encountering this case

Reply:
I think empty_sym is better, since that's not a valid file name.